starfish/imagestack/imagestack.py

```diff
@@ -1121,3 +1159,37 @@ def un_squeeze(self, stack):
         new_shape = (self.num_rounds, self.num_chs, self.num_zlayers) + self.tile_shape
         res = stack.reshape(new_shape)
         return res
+
+
+class _StripArguments:
+    """
+    Class to strip out arguments to an argument call.  This is spiritually the opposite of
+    functools.partial.  When this proxy is called, it removes some arguments and passes it to the
```

Comment:
define "it" here -- the stripped method?

Reply:
Will reword.